addons/frag/functions/fnc_dev_debugAmmo.sqf

```diff
@@ -1,81 +1,103 @@
 #define DEBUG_MODE_FULL
 #include "..\script_component.hpp"
 /*
- * Author: ACE-Team
- *
+ * Author: ACE-Team, Lambda.Tiger
+ * This function will dump every ammo config that would generate ace_frag
+ * fragements that could be fired from a weapon
  *
  * Arguments:
- * None
+ * 0: Log ammo types that wouldn't normall frag
+ * 1: Only export ammo classes of classes referenced in cfgAmmo
+ *    and their submunitions.
+ * 2: Force a CSV format
  *
  * Return Value:
  * None
  *
  * Example:
- * call ace_frag_fnc_dev_debugAmmo
+ * [] call ace_frag_fnc_dev_debugAmmo
  *
  * Public: No
  */
-
 params [
-    ["_debugMissing", true, [false]],
     ["_debugForce", false, [false]],
-    ["_debugSkippedFragPower", 30, [0]]
+    ["_onlyShotAmmoTypes", false, [false]],
+    ["_csvFormat", false, [false]]
 ];
 
 diag_log text format ["~~~~~~~~~~~~~Start [%1]~~~~~~~~~~~~~", _this];
+if (_csvFormat) then {
+    diag_log text format ["ammo,gurney_c,gurney_m,gurney_k,gurney_gC,fragTypes,fragCount,Inheritance"];
+};
 
-private _allMagsConfigs = configProperties [configFile >> "CfgMagazines", "isClass _x", true];
-private _processedCfgAmmos = [];
+// Gather all configs, either those that could be created from firing or all classes
+private _allAmmoConfigs = [];
+if (_onlyShotAmmoTypes) then {
+    private _searchFunc = {
+        params [
+            ["_ammo", "", [""]]
+        ];
+        if (_ammo isEqualTo "" || {_ammo in _allAmmoConfigs}) exitWith {};
+        _allAmmoConfigs pushBack _ammo;
+        private _subMunit = toLower getText (configFile >> "cfgAmmo" >> _ammo >> "submunitionAmmo");
+        if (_subMunit isNotEqualTo "") then {
+            _subMunit = getArray (configFile >> "cfgAmmo" >> _ammo >> "submunitionAmmo");
+            for "_i" from 0 to count _subMunit - 1 do {
+                if (_i mod 2 == 0) then {
+                    [toLower (_subMunit#_i)] call _searchFunc;
```

Comment:
```suggestion
                    [toLowerANSI (_subMunit#_i)] call _searchFunc;
```

Reply:
Addressed in https://github.com/acemod/ACE3/pull/9728/commits/3f45f7735b6b6c50f01d6fcb50db08f20548c0c4